Cannot find module 'lodash.merge' (Error)

So I'm having issues since a few hours now, whenever I try to run `npm run build` (which is effectively `eleventy`) I get 

>Problem writing Eleventy templates: (more in DEBUG output)
> Cannot find module \'lodash.merge\' (Error):
    Error: Cannot find module 'lodash.merge'
        at Function.Module._resolveFilename (internal/modules/cjs/loader.js:581:15)
        at Function.Module._load (internal/modules/cjs/loader.js:507:25)
        at Module.require (internal/modules/cjs/loader.js:637:17)
        at require (internal/modules/cjs/helpers.js:22:18)
        at Object.<anonymous> (/Users/max/mx41/blog/mx41.github.com/node_modules/@11ty/eleventy/src/Engines/JavaScript.js:4:21)
        at Module._compile (internal/modules/cjs/loader.js:689:30)
        at Object.Module._extensions..js (internal/modules/cjs/loader.js:700:10)
        at Module.load (internal/modules/cjs/loader.js:599:32)
        at tryModuleLoad (internal/modules/cjs/loader.js:538:12)
        at Function.Module._load (internal/modules/cjs/loader.js:530:3)
Processed 0 files in 3.34 seconds
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! mx41.github.com@1.0.0 build: `eleventy`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the mx41.github.com@1.0.0 build script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/max/.npm/_logs/2019-01-11T18_50_10_425Z-debug.log`

I thought it is an issue with my node / npm setup, but after uninstalling and reinstalling node several times in different ways (homebrew, manual, now settled with nvm) I am still getting this issue.

I have the slight feeling that it has something to do with my dev machine, as I didn't find anyone else having the problem of missing dependencies.

It should be enough to have @11ty/eleventy as the only dependency for starting out my project, am I right?

--- a/src/Engines/JavaScript.js
+++ b/src/Engines/JavaScript.js
@@ -1,7 +1,5 @@
 const TemplateEngine = require("./TemplateEngine");
 const TemplatePath = require("../TemplatePath");
-const dependencyTree = require("dependency-tree");
-const lodashMerge = require("lodash.merge");
 
 class JavaScript extends TemplateEngine {
   normalize(result) {

